projects/plugins/jetpack/tests/action-test-php.sh

```diff
@@ -2,18 +2,27 @@
 
 set -eo pipefail
 
-echo "::group::Jetpack tests"
-phpunit
-echo "::endgroup::"
-
-if [[ "$WP_BRANCH" == "trunk" ]]; then
-	echo "::group::Jetpack multisite tests"
-	WP_MULTISITE=1 phpunit -c tests/php.multisite.xml
+# Conditionally allow WooCommerce/Jetpack integration tests to run.
+# Unlike WP_MULTISITE or LEGACY_FULL_SYNC, this only enables some tests, it doesn't change the behavior of any.
+if [[ "$WITH_WOOCOMMERCE" == true ]]; then
```

Comment:
Is this enough here? Maybe we should check for the dir too, as there's a chance `$WITH_WOOCOMMERCE` to be truthful but an error occurred while downloading Woo

Reply:
IMO this is fine. If an error occurred while downloading Woo, it should have errored out then.